src/components/SideDiv.js

```diff
@@ -0,0 +1,28 @@
+// SideDiv.js
```

Comment:
Nice bit of custom navigation here! This is a fantastic idea, however, this component should be a child of `App` not its sibling.